modules/ROOT/pages/index.adoc

```diff
@@ -3,4 +3,43 @@
 
 == Introduction
 
-This course covers the installation, updates and uninstallation process for Red Hat Developer Hub (RHDH). You also learn about the basics of configuration in RHDH, and how to configure authentication using a GitHub or GitLab backend along with single sign-on functionality using Keycloak.
\ No newline at end of file
+Welcome to this quick course on the _Red Hat Developer Hub Administration_.
+This course is the `second` in a series of `three` courses about Red Hat Developer Hub:
+
+1. https://redhatquickcourses.github.io/devhub-intro[Introduction to Red Hat Developer Hub^]
+2. Red Hat Developer Hub Administration  (_This course_)
+3. Building Developer Portals with Red Hat Developer Hub (Under Development)
+
+This course covers the installation, updates and uninstallation process for Red Hat Developer Hub (RHDH). You also learn about the basics of configuration in RHDH, and how to configure authentication using a GitHub or GitLab backend along with single sign-on functionality using Keycloak.
+
+== Contributors
+
+The PTL team acknowledges the valuable contributions of the following Red Hat associates:
+
+* Ravi Srinivasan
+* Jason Froelich
```

Comment:
Typo in my last name.  Should be Froe**h**lich.  Missing the `h` after the `e`

Reply:
Sorry! fixed.